Sdk/Track/Swrve.h

```diff
@@ -2,19 +2,18 @@
 #import "SwrveInterfaceOrientation.h"
 #import "SwrveReceiptProvider.h"
 #import "SwrveResourceManager.h"
-#import "SwrveSignatureProtectedFile.h"
-#import "SwrveCommon.h"
 
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wgnu"
+#if COCOAPODS
```

Comment:
The COCOAPODS flag?  SwiftProjectCocoaPods breaks if you try and reference them directly with `#import "SwrveSignatureProtectedFile.h"`, because they specifically are only available via Swift `Modules`.  Swapping the import statements to the proper angular reference, via their framework / module instead `#import <SwrveSDKCommon/SwrveSignatureProtectedFile.h>`, fixing it all up.